code/modules/reagents/newchem/newchem_procs.dm

```diff
@@ -8,17 +8,13 @@
 	var/overdosed = 0 // You fucked up and this is now triggering it's overdose effects, purge that shit quick.
 	var/current_cycle = 1
 
-/datum/reagents
-	var/chem_temp = 300
-	var/list/datum/reagent/addiction_list = new/list()
-
 /datum/reagents/proc/metabolize(mob/living/M)
 	if(M)
 		chem_temp = M.bodytemperature
 		handle_reactions()
 	for(var/A in reagent_list)
 		var/datum/reagent/R = A
-		if(!istype(R))
+		if(!istype(R)) // How are non-reagents ending up in the reagents_list?
```

Comment:
I'd really like to remove this check, but it *was* runtiming in the past--ideally, the culprit of non-reagents winding up in the reagent list should be found, 'cause I have a feeling it's a very rare edge-case.